src/cozmo_companion/assistant.py

```diff
@@ -150,17 +71,34 @@ def __init__(self):
         # Setting up the chatbot with description and tools
         self.chatbot = Application(
             name="Companion",
-            instructions=(
-                "A friendly, supportive chatbot."
-                "It always provides an empathetic response when it detects"
-                "negative sentiment."
-            ),
-            tools=[get_feedback_inquiry],
+            model=config("MARVIN_CHAT_COMPLETIONS_MODEL"),
+            instructions="""A friendly, supportive chatbot.It always provides an
+                empathetic response when it detects negative sentiment. In
+                addition, you have a crucial goal to keep track of the user's
+                sentiment state, especially negative sentiment, to provide more
+                emotionally and contextually aware responses. As an example,
+                the user may have specific requests during an interaction after
+                expressing negative sentiment. For instance, if the last sentiment
+                from the user was negative, and the user asks for a joke, motivational
+                quote, or picture in the next interaction, the chatbot should provide
+                a feedback inquiry based on the user's request type, checking if the
+                joke, quote or picture helped cheer the user up. Note that if the user
+                asks for a picture, you will need to use the tool 'send_picture_to_user'
```

Comment:
Does this work well?

Maybe we can focus on setting up some related tests to evaluate how good the prompts are?

These sentence could also be simplified to reduce grammatical and other complexity for the chat bot. Not sure what is best.